src/Microsoft.ML.DataView/VBufferEditor.cs

```diff
@@ -47,26 +51,16 @@ public static class VBufferEditor
             ref VBuffer<T> destination,
             int newLogicalLength,
             int? valuesCount = null,
+            int? maxValuesCapacity = null,
```

Comment:
>maxValuesCapacity [](start = 17, length = 17)

maximumValuesCapacity?

Reply:
I don't think `max` has to be spelled out to `maximum`. There are plenty of APIs in the BCL that just use "max". Here are some examples from the Http APIs:

https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/ref/System.Net.Http.cs#L93-L96

```C#
        public int MaxAutomaticRedirections { get { throw null; } set { } }
        public int MaxConnectionsPerServer { get { throw null; } set { } }
        public long MaxRequestContentBufferSize { get { throw null; } set { } }
        public int MaxResponseHeadersLength { get { throw null; } set { } }
```